datafusion/core/src/datasource/physical_plan/parquet/mod.rs

```diff
@@ -741,7 +741,18 @@ impl ExecutionPlan for ParquetExec {
     }
 
     fn statistics(&self) -> Result<Statistics> {
-        Ok(self.projected_statistics.clone())
+        // When filters are pushed down, we have no way of knowing the exact statistics.
+        // Note that pruning predicate is also a kind of filter pushdown.
+        // (bloom filters use `pruning_predicate` too)
+        let stats = if self.pruning_predicate.is_some()
+            || self.page_pruning_predicate.is_some()
+            || (self.predicate.is_some() && self.pushdown_filters())
+        {
+            self.projected_statistics.clone().to_inexact()
```

Comment:
Shouldn't we make this absent, unless we actually estimate it using pushed down filters? 🤔

Reply:
I think your idea makes sense, but @alamb  seems to prefer using inexact.

Reply:
I thought that inexact was the least difference than we had

In terms of actually doing cardinality estimation of filters, I think that is its own very hard challenge -- there is some vestigal stuff in DataFusion, but if we wanted to make it better I think we should add some very simple implementation in DataFusion with an API to extend if/when needed